packages/scrollable-image/src/build-code/index.js

```diff
@@ -0,0 +1,72 @@
+import genUniqueId from '../utils/id-generator'
+import get from 'lodash/get'
+import path from 'path'
+import { packageName, urlPrefix } from '../constants'
+
+const _ = {
+  get,
+}
+
+/**
+ *
+ *
+ * @export
+ * @param {Object} - config
+ * @param {Object} - webpackAssets
+ * @params {string} - env
+ * @returns {string} - html string
+ */
+export function buildEmbeddedCode(config, webpackAssets, env = 'production') {
+  const uniqueId = `${packageName}-${genUniqueId()}`
+  const data = _.get(config, 'data')
+  const stringifyData =
+    Array.isArray(data) && data.length > 0
+      ? data.reduce((acc, cur, index) => {
+          if (index === 0) {
+            return `["${cur}"`
+          }
+          if (index === data.length - 1) {
+            return `${acc}, "${cur}"]`
+          }
+          return `${acc}, "${cur}"`
+        }, '')
+      : '[]'
```

Comment:
use `serialize-javascript` to `serialize(data)`, rather than stringify it by ourselves?